.github/workflows/python-release.yml

```
@@ -1,61 +1,61 @@
-      - name: Download validated release wheels
-        uses: actions/download-artifact@3e5f45b2cfb9172054b4087a40e8e0b5a5461e7c # v8.0.1
-        with:
-          name: release-wheels
-          path: dist
-
-      - name: Publish wheels through PyPI Trusted Publishing
-        uses: pypa/gh-action-pypi-publish@ba38be9e461d3875417946c167d0b5f3d385a247 # v1.14.1
-        with:
-          packages-dir: dist/
-          print-hash: true
+# Job-level timeout-minutes so unbounded jobs do not surface to the
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)